### PR TITLE
Fix incorrect AccountPrefix bug and reinstate akash tests

### PR DESCRIPTION
**Pull request by colin-axner**

### CODE REVIEW

**Comment by colin-axner** [comment]
I opened an issue on the [sdk](https://github.com/cosmos/cosmos-sdk/issues/8332)

**Comment by colin-axner** [comment]
I don't think the akash test goes into affect until after this is merged, tested locally that it passes
```
--- PASS: TestAkashToGaiaStreamingRelayer (46.21s)
PASS
ok  	github.com/cosmos/relayer/test	46.257s
```

File: relayer/msgs.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+// the currently set config file. Querying a counterparty would
+// swap the config file. 'MustGetAddress' sets the config file
+// correctly. Do not change this ordering until the SDK config
+// file handling has been refactored.
+// https://github.com/cosmos/cosmos-sdk/issues/8332
+
+// UpdateClient creates an sdk.Msg to update the client on src with data pulled from dst
+func (c *Chain) UpdateClient(dstHeader ibcexported.Header) sdk.Msg {
+	if err := dstHeader.ValidateBasic(); err != nil {
+		panic(err)
```
Comment (fedekunze) [inline relayer/msgs.go:25]: can we return an error instead?

**Review by fedekunze — APPROVED** [review]
utACK. One minor comment and if you can fix a few of the lint issues too

File: relayer/msgs.go (hunk 1 above)
Comment (colin-axner) [reply relayer/msgs.go:25]: this was the existing structure. I'd rather not change too much in one pr

**Comment by colin-axner** [comment]
@fedekunze is there lint issues introduced by new code in this pr? Otherwise I'd rather let it be, there's a long list of existing lint issues and I'd rather spend time on other issues

**Comment by colin-axner** [comment]
I'm going to merge so folks can get back to testing the relayer. Any other requests can be done in followups and ideally would have an issue created for it

File: cmd/raw.go [hunk 2]
```diff
@@ -70,7 +70,7 @@ $ %s tx raw uc ibc-0 ibc-1 ibconeclient`, appName, appName)),
 			if err != nil {
 				return err
 			}
-			return sendAndPrint([]sdk.Msg{chains[src].PathEnd.UpdateClient(updateHeader, chains[src].MustGetAddress())},
```
Comment (AdityaSripal) [inline cmd/raw.go:73]: I like this cleanup!

File: relayer/msgs.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	transfertypes "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
+	conntypes "github.com/cosmos/cosmos-sdk/x/ibc/core/03-connection/types"
+	chantypes "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
+	ibcexported "github.com/cosmos/cosmos-sdk/x/ibc/core/exported"
+)
+
+// NOTE: we explicitly call 'MustGetAddress' before 'NewMsg...'
```
Comment (AdityaSripal) [inline relayer/msgs.go:12]: Thanks for the NOTE

**Pull request merged**